styles/globals.css

```diff
@@ -28,6 +33,42 @@
     border-top-color: black;
 }
 
+.oswald{ 
+    font-family: 'Oswald';
+}
+
+.work-sans{ 
+    font-family: 'Work Sans';
+}
+
+.raleway{
+    font-family: 'Raleway';
+}
+
+.montserrat{
```

Comment:
Not a big deal but to keep styling consistent, put a space between class name and bracket?